reference/constraints/Week.rst

```diff
@@ -0,0 +1,170 @@
+Week
+====
+
+.. versionadded:: 7.2
+
+    The ``Week`` constraint was introduced in Symfony 7.2.
+
+Validates that a string (or an object implementing the ``Stringable`` PHP interface)
+matches a given week number. The week number format is defined by ISO-8601
+and should be composed of the year and the week number, separated by a hyphen
+(e.g. ``2022-W01``).
+
+==========  =======================================================================
+Applies to  :ref:`property or method <validation-property-target>`
+Class       :class:`Symfony\\Component\\Validator\\Constraints\\Week`
+Validator   :class:`Symfony\\Component\\Validator\\Constraints\\WeekValidator`
+==========  =======================================================================
+
+Basic Usage
+-----------
+
+If you wanted to ensure that the ``startWeek`` property of an ``OnlineCourseDTO``
+class is between the first and the twentieth week of the year 2022, you could do
+the following:
+
+.. configuration-block::
+
+    .. code-block:: php-attributes
+
+        // src/Entity/OnlineCourseDTO.php
+        namespace App\Entity;
+
+        use Symfony\Component\Validator\Constraints as Assert;
+
+        class OnlineCourseDTO
```

Comment:
entity and dto seems weird 
what about removing dto ?

Reply:
Agree :+1: